src/util/message.h

```diff
@@ -44,10 +45,47 @@ struct messaget
     template <typename... Args>
     static void println(FILE *f, VerbosityLevel lvl, Args &&...args)
     {
-      if(lvl == VerbosityLevel::Error)
-        fmt::print(f, "ERROR: ");
-      fmt::print(f, std::forward<Args>(args)...);
-      fmt::print(f, "\n");
+    	     
+      	/** 
+		* Newly added code to address issue #886.
+			* It prepends a well formatted timestamp, file path, function name, line number and log level to each LOG message. 
+			* It is transparent to the existing logging function, i.e., log_message.
+			* TODO: It can be implemented using a separate function and invoked here and/or elsewhere when required. 		
+		*/		
+		//========Start===============================================================================================
+		struct tm* time_struct; //A time structure consisting of 9 members, such as tm_sec, tm_min, tm_hour, tm_mday, tm_mon, tm_year, etc. 
+		char time_info[20]; //A variable to hold the date and time values.
```

Comment:
You can define a constexpr string for the format and use its size to allocate the right time size, and as an argument in the strftime call.